Common/src/main/resources/assets/tipsmod/lang/en_us.json

```diff
@@ -70,6 +70,18 @@
   "tipsmod.tip.freezing_leather": "Leather armor can protect against freezing damage.",
   "tipsmod.tip.azalea_caves": "Azalea trees grow above the lush cave biomes.",
 
+  "tipsmod.tip.joke.button": "Despite the name, you don't have to sit on Buttons for them to work",
```

Comment:
Make sure each one ends with a period. The combat, swim, and creeper tip are also missing periods.